src/opensynth/models/faraday/new_gmm/train_gmm.py

```diff
@@ -58,6 +58,8 @@ def training_loop(
     converged = False
     lower_bound = -np.inf
 
```

Comment:
This is just a script that we're explicitly performing e-step/ m-step. I'd add a #TODO here that we'll delete this as part of